docs/sources/whatsnew/whats-new-in-v7-4.md

```diff
@@ -91,7 +91,7 @@ The main use case is for [multi-dimensional](https://grafana.com/docs/grafana/la
 
 ### Alert notification query label interpolation
 
-You can now provide detailed information to alert notification recipients by injecting alert label data as template variables into an alert notification. Labels that exist from the evaluation of the alert query can be used in the alert rule name and in the alert notification message fields using the `${Label}` syntax. The alert label data is automatically injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated.
+You can now provide detailed information to alert notification recipients by injecting alert label data as template variables into an alert notification. Labels that exist from the evaluation of the alert query can be used in the alert rule name and in the alert notification message fields using the `${Label}` syntax. The alert label data is automatically injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated. For more information, refer to the [alert notification docs]({{< relref "../alerting/notifications.md#notification-templating" >}}). 
```

Comment:
Please put the link sentence below the image. This will make it consistent with the other entries.